app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java

```diff
@@ -74,17 +74,17 @@ public Game getGame(){
     }
 
 
-   /* @Rule
-    public ActivityScenarioRule<GameLobbyActivity> testRule = new ActivityScenarioRule<>(GameLobbyActivity.class);
-*/
+    /* @Rule
+     public ActivityScenarioRule<GameLobbyActivity> testRule = new ActivityScenarioRule<>(GameLobbyActivity.class);
+ */
     @Test
     public void activityStartsProperly() {
```

Comment:
I know that this has been here before but might as well get rid of this test since the code inside is commented

Reply:
done